app/controllers/ctc/portal/w2s/employer_info_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Ctc::Portal::W2s::EmployerInfoController < Ctc::Portal::BaseIntakeRevisionController
+  def edit
+    @form = form_class.from_w2(current_model)
```

Comment:
@alaroia-cfa feel free to see if `from_w2` can be e.g. `from_record` so the `#edit` here can go away